llama_index/query_engine/multi_modal.py

```diff
@@ -153,6 +179,30 @@ def _query(self, query_bundle: QueryBundle) -> RESPONSE_TYPE:
 
         return response
 
+    def image_query(self, image_query_str: QueryType, query_str: str) -> RESPONSE_TYPE:
```

Comment:
why does image_query() need both a image_query_str and query_str ? Should image_query_str be optional?

Reply:
And then if not provided, synthesis and retrieval both use the query_str?

Reply:
`image_query_str` is the input image path which is required for image_query. Since our input is a image.
`query_str` is the prompt for sythesis the final results, which is also required for the prompting LLM purpose
let me change the name to avoid confusion